enterprise/lms_api.py

```diff
@@ -24,36 +28,97 @@
 except ImportError:
     CourseEnrollment = None
 
+try:
+    from openedx.core.lib.token_utils import JwtBuilder
+except ImportError:
+    JwtBuilder = None
+
 
+LOGGER = logging.getLogger(__name__)
 LMS_API_DATETIME_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
 
 
 class LmsApiClient(object):
     """
     Object builds an API client to make calls to the edxapp LMS API.
+
+    Authenticates using settings.EDX_API_KEY.
     """
 
-    def __init__(self, url, append_slash=False):
+    API_BASE_URL = settings.LMS_ROOT_URL + '/api/'
```

Comment:
I do agree that some standard ways to reference the API URLs would be great, since I'm sure enterprise is not the only service that wants to talk to these endpoints.

And edx-platform is the logical, central location, since it manages all its service URLs.  But I'm not sure that just a straight move from our `API_BASE_URL` values into edx-platform env is the way to go.

I don't know much about the REST API approach and plans for improvements though.  Are there any plans to standardise the REST APIs?  Each individual sub-service seems to provide its own path, with no standard way to expose the path, and the currently supported version number.

So no suggestions offhand..  Would need to do some more digging into how the other services access the APIs, and to see what edX's plans are.